cli/slsa-verifier/main.go

```diff
@@ -64,17 +79,18 @@ func main() {
 		os.Exit(1)
 	}
 
-	verifiedProvenance, _, err := runVerify(artifactPath, provenancePath, source,
-		pbranch, pbuilderID, ptag, pversiontag)
+	verifiedProvenance, _, err := runVerify(artifactImage, artifactPath, provenancePath,
+		source, pbranch, pbuilderID, ptag, pversiontag)
 	if err != nil {
 		fmt.Fprintf(os.Stderr, "FAILED: SLSA verification failed: %v\n", err)
 		os.Exit(2)
 	}
 
 	fmt.Fprintf(os.Stderr, "PASSED: Verified SLSA provenance\n")
-
 	if printProvenance {
-		fmt.Fprintf(os.Stdout, "%s\n", string(verifiedProvenance))
+		for _, verified := range verifiedProvenance {
```

Comment:
This won't be a valid JSON output anymore, will it?
I wonder if we should simplify and print a single valid provenance. Later I agree we need t output all, but we need to decide the format. Wdut?

Reply:
Note: the code currently prints one character per line, I think

Reply:
Oh no! Good point. This should have been one provenance each. Will simplify to a single valid provenance.